code/modules/client/preference_setup/loadout/loadout_general.dm

```diff
@@ -3,8 +3,12 @@
 	path = /obj/item/weapon/cane
 
 /datum/gear/dice
-	display_name = "d20"
-	path = /obj/item/weapon/dice/d20
+	display_name = "pack of dices"
```

Comment:
This should just be "dice".

Reply:
It is more than one.

Reply:
`dice` is plural. `dices` is not a word.